genai-on-vertex-ai/vertex-ai-search/Ingesting_unstructured_docuemtns_with_metadata.ipynb

```diff
@@ -0,0 +1,1054 @@
+{
```

Comment:
<p>Separate out Colab Auth as it won't work in Workbench insgtances.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Updated Auth blocks, PTAL.</p>